### PR TITLE
add tv4 v1.2.7 with autoupdate config

### LINKED ISSUE #7884: [Request] Add tv4.js
**Library name:** tv4.js
**Git repository url:**
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dannyxx001**
PR for #7884
@pvnr0082t please help me review this PR, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/geraintluff/tv4
 * Official website (optional, not the repository): https://geraintluff.github.io/tv4/
 * NPM package url (optional): https://www.npmjs.com/package/tv4
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 712
   - Count of watchers: 36
   - Count of forks: 118
 * NPM download stats (optional):
   - Downloads in the last day: 65,444
   - Downloads in the last week: 373,222
   - Downloads in the last month: 1,683,132

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7884, cc @geraintluff

### CODE REVIEW

File: ajax/libs/tv4/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "validator",
+    "tv4"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/geraintluff/tv4.git"
+  },
+  "homepage": "https://geraintluff.github.io/tv4/",
+  "licenses": [
+    "Public Domain",
```
Comment (pvnr0082t) [inline ajax/libs/tv4/package.json]: @dannyxx001 `Public Domain` doesn't exist in [license-list](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json). You need to provide `url` for the license.

Comment (dannyxx001) [inline ajax/libs/tv4/package.json]: It should be one line or not?
For example,
this
{
  "type": "Public Domain",
  "url": "http://geraintluff.github.io/tv4/LICENSE.txt"
}
or this
"Public Domain http://geraintluff.github.io/tv4/LICENSE.txt"

Comment (pvnr0082t) [inline ajax/libs/tv4/package.json]: The former

**Comment by dannyxx001** [comment]
@pvnr0082t I have updated it. Please help me review again, thank you.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
Don't we need `lang/`?

**Comment by dannyxx001** [comment]
@pvnr0082t I have updated it. Please help me review again, thanks.

**Comment by PeterDaveHello** [comment]
Why don't you minify language files?

**Comment by dannyxx001** [comment]
@pvnr0082t I have minified them noticed by Peter. Please help me review again, thanks.

**Comment by pvnr0082t** [comment]
@dannyxx001 Your [pl-PL.js](https://github.com/dannyxx001/cdnjs/blob/b52c367520b781728cef37d11393774bb364167d/ajax/libs/tv4/1.2.7/lang/pl-PL.js) file is an executable file. Please modify it to unexecutable.

**Comment by dannyxx001** [comment]
OK. Done. Please help me review again, thank you.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Pull request merged**